lms/static/sass/discussion/elements/_actions.scss

```diff
@@ -160,7 +160,7 @@
     }
 
     &.action-vote {
-
+       display: inline-block !important;
```

Comment:
@adampalay You mean any other way to achieve this? 

@attiyaIshaque Does this really fixes the issues?